mne/icons/mne.qrc

```diff
@@ -10,5 +10,6 @@
     <file alias="clear.svg">clear-black-18dp.svg</file>
     <file alias="screenshot.svg">screenshot-black-18dp.svg</file>
     <file alias="movie.svg">movie-black-18dp.svg</file>
+    <file alias="mne-icon.png">mne-circle-black.png</file>
```

Comment:
@GuillaumeFavelier let me know if this fits into the existing Qt resource scheme well enough

Reply:
This is good enough :+1: